### PR TITLE
Add an easter egg in the newspaper

### PR DESCRIPTION
**Pull request by BrettDong**
Add three news reports.

Basically, the plot is that I was a victim of a failed secret teleportation experiment and I was teleported to an unknown place, maybe in another city, or in another continent, or even into another universe.

Please point out any mistake as I am not a native English speaker, thanks.

### CODE REVIEW

File: data/json/snippets.json [hunk 1]
```diff
@@ -1464,7 +1464,10 @@
       "GLAMOPOLITAN!  We've got ALL the latest tips!  Whether you want to know what the elite are eating, wearing or discussing, Glamopolitan is YOUR magazine!  So pick up a copy today and \"Sizzle Like A Star\"!",
       "POPULAR MECHANICS: People say mechanics is boring.  We say, Prove them Wrong!  We've got all the articles that make it interesting to talk about, so you can \"Make Mechanics Popular\"!",
       "CRAFTY CRAFTERS QUARTERLY: Macaroni isn't just for eating anymore!  Learn how to make jewelery and art from it as well!  We also discuss the correct way to use superglue without gluing your hands together!",
-      "UNDERGROUND BUNKER?  Sources in the government suggest that a underground bunker is being built in a secret location in case of enemy attack.  We were unable to get the exact location as our sources mysteriously disappeared shortly after revealing this information."
+      "UNDERGROUND BUNKER?  Sources in the government suggest that a underground bunker is being built in a secret location in case of enemy attack.  We were unable to get the exact location as our sources mysteriously disappeared shortly after revealing this information.",
```
Comment (Leland) [inline data/json/snippets.json]:
```diff
- a underground bunker
+ an underground bunker
```

File: data/json/snippets.json [hunk 2]
```diff
@@ -1464,7 +1464,10 @@
...
+      "STUDENT MISSING: A high school student vanished yesterday evening in the forest near Wayland.  The 17-year-old international student from China, who adopted an anglicized nickname Brett for Americans always mispronouncing his Asian name, was last with his friends in the camp.  \"He said he's gonna fetch some wood, then we'd never seen him back[..]\" says his classmate Jianxiang Wang.  The search is underway.",
```
Comment (Leland) [inline data/json/snippets.json]: This is how I'd do this:

> The 17-year-old international student from China (who adopted the anglicized nickname "Brett" due to Americans difficulty with his given name) was last seen with his friends in the camp

Also:

```diff
- \"He said he's gonna fetch some wood, then we'd never seen him back[..]\" says his classmate
+ \"Brett said that he was gonna get some firewood but he never came back,\" said his classmate, Jianxiang Wang.
```

Comment (BrettDong) [inline data/json/snippets.json]: Thanks for the feedback!

**Comment by BrettDong** [comment]
@Qrox Is it okay to write your name in the newspaper?

**Comment by Qrox** [comment]
@BrettDong sure, you can do it.

**Comment by BrettDong** [comment]
@Leland I updated the sentences. Are they looking correct now?

**Comment by Leland** [comment]
@BrettDong I noticed some other grammatical issues. To reviewers, hold on merging this till I get another chance to go over them.

File: data/json/snippets.json [hunk 3]
```diff
@@ -1464,7 +1464,10 @@
...
+      "UNDERGROUND BUNKER?  Sources in the government suggest that an underground bunker is being built in a secret location in case of enemy attack.  We were unable to get the exact location as our sources mysteriously disappeared shortly after revealing this information.",
+      "STUDENT MISSING: A high school student vanished yesterday evening in the forest near Wayland.  The 17-year-old international student from China (who adopted the anglicized nickname \"Brett\" due to Americans' difficulty pronouncing his given name) was last seen with his friends in the camp.  \"Brett said that he was gonna get some firewood but he never came back,\" said his classmate, Jianxiang Wang.  The search is underway.",
+      "STILL SEARCHING: The search for Brett, the high school student went missing three days ago, is still ongoing.  The County Search & Rescue was brought in, however, the missing midfielder for his school soccer team had still not been located as of writing this report.  \"He could have played in the match yesterday,\" says sorrowfully his teammate, \"[..] we've never stopped praying.\"",
```
Comment (Leland) [inline data/json/snippets.json]:
```diff
-high school student went missing three days ago
+high school student who went missing three days ago
```
I'd probably reword the second part of this snippet like so:

> "STILL SEARCHING: The search for Brett, the high school student who went missing three days ago, is still ongoing.  \"He could have played in the soccer game yesterday,\" Brett's sorrowful teammate said, \"[..] we've never stopped praying.\" Despite the best efforts of the County Search & Rescue, Brett had still not been located at the time of this report.",

File: data/json/snippets.json [hunk 4]
```diff
@@ -1464,7 +1464,10 @@
...
+      "RUMORS DENIED: Rumors had been swirling since mid-July about the military conducting secret teleportation experiments in an underground research facility near Wayland.  \"I have to clarify is that we'd never done such research,\" says the officer on the press conference, \"Teleportation only exists in sci-fi films.\""
```
Comment (Leland) [inline data/json/snippets.json]: I'm definitely going overboard editing an easter egg in a snippet but whatever: this snippet only mentions a press conference indirectly ("says the officer on the press conference") - it'd be better to mention that outright. Maybe something like this:

> "RUMORS DENIED: Allegations that the military had been conducting teleportation experiments in secret just outside Wayland were put to rest during a press conference earlier today.  \"I have to clarify that not only have we never done such research,\" said the officer on the press conference, \"teleportation only exists in sci-fi films.\""

**Comment by BrettDong** [comment]
@Leland Thank you very much for your help. I still have tons to learn about the English language. 😆

**Pull request merged**